resources/css/app.css

```
@@ -149,84 +149,80 @@
+    --muted: #f1f5f9;
+    --muted-foreground: #64748b;
+    --accent: #c4a5f3;
+    --accent-foreground: #0f172a;
     --destructive: #dc2626;
-    --border: #e2e0d8;
+    --border: #f1f5f9;
     --input: #ffffff;
-    --ring: #18181b;
+    --ring: #c4a5f3;
     --radius: 0.25rem;
     color-scheme: light;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)